src/components/header.tsx

```diff
@@ -1,109 +1,236 @@
 import React from "react";
-import { IoIosArrowBack } from "react-icons/io";
-
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Drawer from "@mui/material/Drawer";
+import IconButton from "@mui/material/IconButton";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemText from "@mui/material/ListItemText";
+import Toolbar from "@mui/material/Toolbar";
+import MenuIcon from "@mui/icons-material/Menu";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import LaunchIcon from "@mui/icons-material/Launch";
 import {
   advisoryLogo,
   advisoryMentorshipLogo,
   advisoryMentorshipPartners,
 } from "../assets";
 
 import "./header.css";
+const drawerWidth = 240;
 
-const NavBar = () => (
-  <div className="nav">
-    <a className="nav-back-to-main-site" href="https://advisory.sg">
-      <IoIosArrowBack className="nav-arrow-icon" />
-      back to main site
-    </a>
-    <img className="nav-logo" src={advisoryLogo} alt="Advisory" />
-  </div>
-);
+interface Props {
+  window?: () => Window;
+}
+
+const ResponsiveDrawer = (props: Props) => {
+  const { window } = props;
+  const [mobileOpen, setMobileOpen] = React.useState(false);
+
+  const handleDrawerToggle = () => {
+    setMobileOpen(!mobileOpen);
+  };
+
+  const LinkList = () => {
+    return (
+      <div>
+        <List>
+          <ListItem disablePadding>
+            <ListItemButton component="a" href="https://advisory.sg">
+              <ListItemText primary="Back To Main Site" />
+              <ListItemIcon style={{ minWidth: "60px" }}>
```

Comment:
Instead of setting `style` here, could you set the following styles for `ListItemText`:
1. `flex-grow: 0;` (prevents element from growing)
2. `margin-right: 0.5rem;` (sets a custom margin after the text; feel free to change this value)

E.g.
```
              <ListItemText primary="Back To Main Site" style={{
                flexGrow: "0", marginRight: "0.5rem"
              }} />
              <ListItemIcon><LaunchIcon /></ListItemIcon>
```

![image](https://user-images.githubusercontent.com/2070423/161435632-7ae7ebb7-6c1a-4b18-ae1c-c4678ecaf98f.png)